examples/with-nanostores/src/pages/index.astro

```diff
@@ -3,7 +3,6 @@
 import AdminsReact from '../components/AdminsReact.jsx';
 import AdminsSvelte from '../components/AdminsSvelte.svelte';
 import AdminsVue from '../components/AdminsVue.vue';
-import AdminsPreact from '../components/AdminsPreact.jsx';
```

Comment:
This is a change worth calling out... Had to drop the `@nanostores/preact` usage from this example to get everything passing. I think there's some issue with deduping going on, but I'm hoping it's just in our monorepo. Will be sure to check this once we cut a release.

Reply:
Thanks for the callout. If you're worried about not being able to resolve this before end of week, I'd suggest creating an issue to track adding `@nanostores/preact` back.